src/test/scala/nl/knaw/dans/lib/encode/StringEncodingSpec.scala

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.lib.encode
+
+import org.scalatest.{ FlatSpec, Matchers, OptionValues }
+
+class StringEncodingSpec extends FlatSpec with Matchers with OptionValues {
```

Comment:
I think it would be fine to merge these two test files into one. Probably you can even use [`behave like`](http://www.scalatest.org/user_guide/sharing_tests), as these tests are mostly the same...

Reply:
done